### PR TITLE
Ported Pill 5

### PR DESCRIPTION
**Pull request by ixxie**
Ported the fifth pill.

### CODE REVIEW

**Comment by ixxie** [comment]
1. Some variables were quoted ("a"); I removed quotes and added literals to make style consistent.
2. I followed @grahamc by using literals for code but I did noticed there are `<code>` tags; maybe the literals should be replaced at some point.
3. Some paragraphs were ambiguous in that there was a line break in the middle without a gap line. In those cases I removed the break since they seemed like one paragraph rather than two.
4. The link to the functions section in the manual was broken; I had to guess the section intended in the post.
5. The first section missed a title so I added one.

File: pills/05-functions-and-imports.xml [hunk 1]
```diff
@@ -1,8 +1,348 @@
...
-      xml:id="functions-and-imports">
+xmlns:xlink="http://www.w3.org/1999/xlink"
+xmlns:xi="http://www.w3.org/2001/XInclude" version="5.0"
+xml:id="functions-and-imports">
+
+      <title>functions and imports</title>
+
+<section>
+      <title>Functions</title>
+      <para>
```
Comment (grahamc) [inline pills/05-functions-and-imports.xml]: these introductory para's don't need a section just on their own, just "leading paragraphs" or whatever docbook wants to call them.

File: pills/05-functions-and-imports.xml [hunk 2]
```diff
@@ -1,8 +1,348 @@
 <chapter xmlns="http://docbook.org/ns/docbook"
-      xmlns:xlink="http://www.w3.org/1999/xlink"
-      xmlns:xi="http://www.w3.org/2001/XInclude"
-      version="5.0"
-      xml:id="functions-and-imports">
+xmlns:xlink="http://www.w3.org/1999/xlink"
+xmlns:xi="http://www.w3.org/2001/XInclude" version="5.0"
+xml:id="functions-and-imports">
```
Comment (grahamc) [inline pills/05-functions-and-imports.xml:4]: Please restore the formatting to what it was

File: pills/05-functions-and-imports.xml [hunk 3]
```diff
@@ -1,8 +1,348 @@
...
+
+      <para>
+            I remind you how to enter the Nix environment: <code>source
+            ~/.nix-profile/etc/profile.d/nix.sh</code>
+      </para>
+
+</section>
+
+<section>
+      <title>Nameless and single parameter</title>
```
Comment (grahamc) [inline pills/05-functions-and-imports.xml]: To remain consistent with the rest of the doc, can you use 2 space indents, and keep sections indented one level in (as it is a child of the `chapter`)? I've created an editorconfig file to include the indentation information.

**Review by grahamc — COMMENTED** [review]
Overall on a quick check (I ran out of time for a thorough one at this moment) it this looks great, but some indentation changes requested, and please don't add the first paragraphs in to a section.

**Comment by ixxie** [comment]
@grahamc - fixed, and fixed; also ported Pill 6 now. 

Notes:
- I noticed that `<emphasis role="underline">` doesn't seem to work; perhaps some css is needed in conjunction?
- The end of pill 6 has a lovely little inline trollface which I cannot reproduce. I left the code in there since it builds, but it doesn't seem to render.

**Pull request merged**